acai-ios/CollectionViewCells/HeaderImageCollectionViewCell.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  BowlHeaderCollectionViewCell.swift
+//  acai-ios
+//
+//  Created by Artesia Ko on 3/21/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+class HeaderImageCollectionViewCell: UICollectionViewCell {
+    
+    // MARK: View vars
+    var imageView: UIImageView!
+    
+    // MARK: Constraint Constants
+    let imageViewHeightWidth = 206
+    let imageViewTopOffset = 15
```

Comment:
if we declare these as `let imageViewHeightWidth: CGFloat = 206`, we won't have to do the cast later on like you do in line 27